stetho/src/main/java/com/facebook/stetho/inspector/elements/ChainedDescriptor.java

```diff
@@ -172,6 +172,11 @@ public final Object getChildAt(Object element, int index) {
     return onGetChildAt((E)element, thisIndex);
   }
 
+  @Override
+  public void setAttributesAsText(Object element, String text) {
```

Comment:
hmm why doesn't this follow the pattern of all the other methods in this class? e.g. with an `onSetAttributesAsText(E, String)` method? In this case you can follow the pattern set by `getNodeType()` and `onGetNodeType()`: the default implementation of `onGetNodeType()` just calls mSuper's implementation.